sdk/devcenter/azure-developer-devcenter/tests/testcase.py

```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import os
+import functools
+from devtools_testutils import AzureRecordedTestCase, EnvironmentVariableLoader
+from azure.developer.devcenter import DevCenterClient
+
+
+class DevcenterTest(AzureRecordedTestCase):
+    def create_client(self, tenant_id, dev_center):
+        credential = self.get_credential(DevCenterClient)
+        return DevCenterClient(
+            dev_center=dev_center,
+            tenant_id=tenant_id,
+            credential=credential
+        )
```

Comment:
tenant_id shall be devcenter_tenant_id, dev_center shall be devcenter_name which keeps same with 
![image](https://user-images.githubusercontent.com/70930885/196319355-2ebd9efb-d7f4-44cb-a7b2-797eda030284.png)

Reply:
This is an internal function called by the test though, the powershell preparer is not applied to these